### PR TITLE
Add getProductTradeStream method

### PR DESCRIPTION
**Pull request by fb55**

### CODE REVIEW

File: tests/public_client.js [hunk 1]
```diff
@@ -55,3 +55,42 @@ test('public client should return values', function(done) {
     done();
   });
 });
+
+test('public client should stream trades', function(done) {
+  this.timeout(6e3);
```
Comment (mihar) [inline tests/public_client.js]: style: don't use scientific notation

Comment (fb55) [inline tests/public_client.js]: Fixed

**Comment by mihar** [comment]
@fb55 can you resolve the conflicts plz

**Comment by fb55** [comment]
done

**Pull request merged**